src/meta/RootUserMan.h

```diff
@@ -20,6 +20,27 @@ namespace meta {
  * */
 class RootUserMan {
  public:
+  static bool isGodExists(kvstore::KVStore* kv) {
+    auto rolePrefix = MetaKeyUtils::roleSpacePrefix(kDefaultSpaceId);
+    std::unique_ptr<kvstore::KVIterator> iter;
+    auto code = kv->prefix(kDefaultSpaceId, kDefaultPartId, rolePrefix, &iter, false);
+    if (code != nebula::cpp2::ErrorCode::SUCCEEDED) {
+      LOG(INFO) << "Prefix God Role Failed";
+      return false;
```

Comment:
Will god user be created again if this read failed?  Check plz.

Reply:
fix